composer.json

```diff
@@ -30,7 +30,7 @@
         "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation."
     },
     "require-dev": {
-        "phpunit/phpunit": "*",
+        "phpunit/phpunit": "~5.7.0",
```

Comment:
Wouldn't using constraint `@stable` pick the latest stable based on PHP version?